src/main/java/com/cityelf/model/WaterForecast.java

```diff
@@ -0,0 +1,99 @@
+package com.cityelf.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "water_forecasts")
+public class WaterForecast {
+  @Id
+  private long id;
+  @Column(name = "start")
+  private LocalDateTime start;
+  @Column(name = "estimated_stop")
+  private LocalDateTime estimatedStop;
+  @OneToOne
```

Comment:
This is not @OneToOne relationship. It is @ManyToOne, since you can have a lot of the same address_id in the water_forecasts table.